modules/custom/az_paragraphs/az_paragraphs_photo_gallery/templates/paragraph--az-photo-gallery.html.twig

```diff
@@ -57,21 +57,23 @@
         {% if content.field_az_photos[0] %}
         {% if grid %}
           {# Render the images for the gallery grid display. #}
```

Comment:
Were these changes on purpose?

Reply:
NVM I see this now

> Addressed Twig deprecations (see [twig.symfony.com/doc/2.x/deprecated.html#tags](https://twig.symfony.com/doc/2.x/deprecated.html#tags))
Adding an if condition on a for tag is deprecated in Twig 2.10. Use a filter filter or an "if" condition inside the "for" body instead (if your condition depends on a variable updated inside the loop).

Reply:
Yes.  See the updated issue description.